routes/web/project/load_balancer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch(:project_prefix, "load-balancer") do |r|
+    r.get true do
+      lbs = @project.private_subnets.flat_map { |ps| ps.load_balancers_dataset.authorized(@current_user.id, "LoadBalancer:view").all }
```

Comment:
Why don't use `@project.load_balancers_dataset.authorized(@current_user.id, "LoadBalancer:view").all` directly? Since we associate load balancers with the project, and it has separate permissions than private subnets. I think it's better approach.

Reply:
Updated.